src/common/snippets/src/lowered/pass/init_loops.cpp

```diff
@@ -16,7 +16,7 @@ namespace pass {
 using LoopPort = LinearIR::LoopManager::LoopPort;
 
 namespace {
-int64_t get_dim_stride(size_t dim, const std::vector<size_t>& layout, const std::vector<size_t>& shape) {
+int64_t get_input_stride(size_t dim, const std::vector<size_t>& layout, const std::vector<size_t>& shape) {
```

Comment:
Minor: should we use an alias `VectorDims` instead of `std::vector<size_t>` here and in `get_output_stride`?